platform/administer/templates/parameters.mdx

```diff
@@ -0,0 +1,422 @@
+
+```yaml title="Enum with descriptions (resource tier selection)"
+- variable: instance_size
+  label: Instance Size
+  description: Choose the resource allocation tier for your virtual cluster
+  type: string
+  required: true
+  options:
+    - value: small
+      description: 1 CPU, 2GB RAM - suitable for development
+    - value: medium
+      description: 2 CPU, 4GB RAM - suitable for testing
+    - value: large
+      description: 4 CPU, 8GB RAM - suitable for production workloads
+  section: Resource Configuration
+```
+
+#### Conditional parameters
+
+The following example shows how to conditionally display parameters based on user input. The `storage_size` field only appears if `enable_persistence` is set to `true`. It also uses a regular expression to validate that the input follows a specific storage format.
+
+```yaml title="Conditional parameters with regex validation"
+- variable: enable_persistence
+  label: Enable Persistent Storage
+  description: Add persistent volumes to the virtual cluster
+  type: boolean
+  default: false
+  section: Storage Settings
+
+- variable: storage_size
+  label: Storage Size
+  description: Size of persistent storage (e.g., 10Gi, 50Gi)
+  type: string
+  required: true
+  validation: "^[0-9]+[KMGT]i$"
+  condition: "{{ .Values.enable_persistence }}"
+  section: Storage Settings
+```
+
+## Access parameter values in templates
+
+After you define parameters, you can access their values in your template using Go template syntax. The platform makes all parameter values available under the `.Values` object.
+
+### Basic parameter access
+
+```yaml
+apiVersion: vcluster.loft.sh/v1
+kind: VCluster
+metadata:
+  name: "{{ .Values.cluster_name }}"
+  labels: 
+    environment: "{{ .Values.environment_type }}"
+    project: "{{ .Values.project_name }}"
+spec:
+  controlPlane:
+    replicas: {{ .Values.replica_count }}
+```
+
+### Conditional template logic
+
+Use Go template conditionals to create dynamic configurations based on parameter values:
+
+```yaml
+spec:
+  controlPlane:
+    resources:
+      limits:
+        # Different resource limits based on environment
+        cpu: >-
+          {{- if eq .Values.environment_type "production" -}}
+          4000m
+          {{- else if eq .Values.environment_type "staging" -}}
+          2000m
+          {{- else -}}
+          1000m
+          {{- end }}
+        memory: >-
+          {{- if eq .Values.environment_type "production" -}}
+          8Gi
+          {{- else if eq .Values.environment_type "staging" -}}
+          4Gi
+          {{- else -}}
+          2Gi
+          {{- end }}
+  
+  # Conditional features based on boolean parameters
+  {{- if .Values.enable_monitoring }}
+  monitoring:
+    enabled: true
+    prometheus:
+      enabled: true
+    grafana:
+      enabled: true
+  {{- end }}
+  
+  {{- if .Values.enable_persistence }}
+  storage:
+    persistence: true
+    size: "{{ .Values.storage_size }}"
+  {{- end }}
+```
```

Comment:
I think this is intended to be `vcluster.yaml`, but it doesn't line up with the spec.